### ISSUE TITLE
Run Gatsby front-end on `/gatsby` for comparison to Next.js

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#1605]
**What would you like to be added**:

A great suggestion from @cindyledev on Slack: let's run our current Gatsby front-end in parallel to the next.js version on staging/production.  Once we reach 100% parity between next and Gatsby we can remove it.

This is a similar idea to what we do with https://dev.telescope.cdot.systems/planet (i.e., the `/planet` route).  See https://github.com/Seneca-CDOT/telescope/search?q=legacy&type=code for how we do this.

**Why would you like this to be added**:

As we work toward feature parity between the next and Gatsby front-ends, it would be nice to have an easy way to compare the two on the same deployment.

**Comment by humphd** [comment]
We're dropping gatsby, closing.
